src/main/java/teammates/ui/controller/Action.java

```diff
@@ -127,7 +127,11 @@ protected String getRegkeyFromRequest() {
         String regkey = getRequestParamValue(Const.ParamsNames.REGKEY);
         if (regkey == null) {
             //TODO: remove this branch on October 15th 2014.
-            return getRequestParamValue(Const.ParamsNames.REGKEY_LEGACY);
+            String legacyRegkey = getRequestParamValue(Const.ParamsNames.REGKEY_LEGACY);
+            if (legacyRegkey != null) {
+                log.severe("TEAMMATES accessed using old join link");
```

Comment:
I think we can include the key in the log message as mostly we expect the user not to be loggedin. Also at this time, we do not have the student information yet